sample-apps/python-sample-app/random_metrics.py

```diff
@@ -0,0 +1,99 @@
+import random
+import threading
+import time
+
+from config import *
+from opentelemetry.metrics import CallbackOptions, Observation
+from opentelemetry import metrics
+
+cfg = create_config('config.yaml')
+
+meter = metrics.get_meter(__name__)
+
+# Starts the callback for cpu usage
+def cpu_usage_callback(options: CallbackOptions):
+    min = 0
+    max = cfg['random_cpu_usage_upper_bound']
```

Comment:
Update the way you access `cfg` with the camelCase instead of snake_case here and in the other `cfg` usages